scst-store/use-external-database.hbs.md

```diff
@@ -1,43 +1,56 @@
 # Use external PostgreSQL database for Supply Chain Security Tools - Store
 
-This topic describes how you can configure and use your external PostgreSQL database for Supply Chain Security Tools (SCST) - Store.
+For production deployments, it is recommended to use an external Postgres database rather the one packaged with the Tanzu Application Platform.  This will allow you to manage it via best practices and process that your organization have established.  This topic describes how you can configure your TAP deployment to use an external database.
```

Comment:
```suggestion
For production deployments, it is recommended to use an external Postgres database rather the one packaged with the Tanzu Application Platform.  This will allow you to manage it via best practices and processes that your organization have established.  This topic describes how you can configure your TAP deployment to use an external database.
```